includes/wikia/parser/templatetypes/TemplateTypesParser.class.php

```diff
@@ -116,7 +116,7 @@ public static function onGetTemplateDom( $title, $args, $frame, &$outputText ) {
 	 * @return bool
 	 */
 	public static function onEndBraceSubstitution( $templateTitle, &$templateWikitext, &$parser ) {
-		global $wgEnableContextLinkTemplateParsing, $wgEnableInfoIconTemplateParsing;
+		global $wgEnableContextLinkTemplateParsing, $wgEnableInfoIconTemplateParsing, $wgEnableNavigationTemplateParsing;
```

Comment:
@dianafa Is there a reason for this duplicated check? If not I will remove the one located in ``Parser``.